src/NzbDrone.Core/ImportLists/HeadphonesImport/HeadphonesImport.cs

```diff
@@ -10,6 +11,7 @@ public class HeadphonesImport : HttpImportListBase<HeadphonesImportSettings>
         public override string Name => "Headphones";
 
         public override ImportListType ListType => ImportListType.Other;
+        public override TimeSpan MinRefreshInterval => TimeSpan.FromHours(12);
```

Comment:
Given this is another user app, we can bring this down off 12 hours some, 6 or 1